msfdb

```diff
@@ -906,6 +906,46 @@ def invoke_command(commands, component, command)
   end
 end
 
+# Searches the user's path for the executable file that would be run had the
+# command actually been invoked.
+# @param cmd [String] command name
+# @param return_all [Boolean] If true, returns a list of all instances of
+# executable found (instead of just the first one). Default: false.
+# @return [Array] instance or instances of executable found based on return_all, or nil.
+def which(cmd, return_all: false)
```

Comment:
can you improve the 'which' implementation in lib/msf/util/helper.rb rather than creating a new one here? Or is it hard for these to share the same code?

Reply:
I honestly didn't realize there was a 'which' implementation in `lib/msf/util/helper.rb`. I'll take a look.